deployments/hadoop-yarn/ansible/roles/prometheus/vars/main.yml

```
@@ -0,0 +1,6 @@
+---
+serviceName: "prometheus"
+userId: "prometheus"
+groupId: "prometheus"
+exec_command: "/usr/local/bin/prometheus --config.file=/etc/prometheus/prometheus.conf --storage.tsdb.path=/data/prometheus --storage.tsdb.retention=2d"
+version: "2.3.2"
```

Why are we installing an old version ?
https://github.com/prometheus/prometheus/releases
https://github.com/prometheus/prometheus/releases/latest
```suggestion
version: "2.26.0"
```